bike_kollective/lib/data/model/bike.dart

```diff
@@ -20,6 +19,7 @@ class BikeModel {
   final DateTime locationUpdated;
   final List<BKDocumentReference> rides;
   final List<BKDocumentReference> issues;
+  final double? distance; // Optional distance property
```

Comment:
Distance should be derived from the active user's location and the location? This assumes only one user, which isn't true

Reply:
I recommend removing this from the model object and calculating it elsewhere or creating another class BikeWithDistance that is a tuple of a bike and a distance. The model class should only contain data that is intended for the database.